examples/inflating_sheet/pressure_continuation_solver.h

```diff
@@ -36,6 +36,8 @@ namespace GRINS
 
     PressureContinuationSolver( const GetPot& input );
 
+    PressureContinuationSolver() = delete;
```

Comment:
Is this necessary?  The default constructor shouldn't be implicitly defined in classes with a user-defined constructor, right?

Reply:
I did not know this. (I sanity checked on a small repro.) Thanks for pointing it out. Now to slog through and clean all of those up. (The downside of developing everything in the open is my ignorance is on full display and, therefore, the resulting shame is not private.) Really probably ought to do a pass better enforcing rule-of-five, but that's for another PR and another day.